apps/teams-test-app/src/components/RememberScrollToggle.tsx

```diff
@@ -0,0 +1,54 @@
+import throttle from 'lodash.throttle';
+import React, { useEffect, useState } from 'react';
+
+const RememberScrollToggle = () => {
+  // Initialize the toggle state from sessionStorage
+  const [isScrollFeatureActive, setIsScrollFeatureActive] = useState(
+    () => JSON.parse(sessionStorage.getItem('isScrollFeatureActive') || '{}') ?? true,
+  );
+
+  useEffect(() => {
+    // Save the toggle state to sessionStorage whenever it changes
+    sessionStorage.setItem('isScrollFeatureActive', JSON.stringify(isScrollFeatureActive));
+
+    console.log('scrolling');
```

Comment:
Do we want to keep the `console.log` statements in here?

Reply:
Probably not, this looks like it was just used during development to make sure the function was getting called and also to make sure it wasn't getting called too frequently. Sound correct @alexneyman-MSFT ?